tests/unit/workspace_access/test_groups.py

```diff
@@ -132,25 +133,29 @@ def test_snapshot_should_consider_groups_defined_in_conf():
     wsclient = MagicMock()
     group1 = Group(id="1", display_name="de", meta=ResourceMeta(resource_type="WorkspaceGroup"))
     group2 = Group(id="2", display_name="ds", meta=ResourceMeta(resource_type="WorkspaceGroup"))
-    wsclient.groups.list.return_value = [group1, group2]
     acc_group_1 = Group(id="11", display_name="de")
     acc_group_2 = Group(id="12", display_name="ds")
     wsclient.api_client.do.return_value = {
         "Resources": [g.as_dict() for g in [acc_group_1, acc_group_2]],
     }
-    res = GroupManager(backend, wsclient, inventory_database="inv", include_group_names=["de"]).snapshot()
-    assert res == [
-        MigratedGroup(
-            id_in_workspace="1",
-            name_in_workspace="de",
-            name_in_account="de",
-            temporary_name="ucx-renamed-de",
-            members=None,
-            external_id="11",
-            roles=None,
-            entitlements=None,
-        )
-    ]
+
+    gm = GroupManager(backend, wsclient, inventory_database="inv", include_group_names=["de"])
+
+    with patch.object(gm, "_list_workspace_groups", return_value=[group1, group2]):
```

Comment:
don't mock out the code written in this repo - mock out the code that this repo depends upon.